genestack_client/utils.py

```diff
@@ -89,6 +89,9 @@ def get_user(args=None):
         args = make_connection_parser().parse_args()
 
     alias = args.user
+    if not alias and args.token:
+        return User(email=None, host=args.host, password=None, token=args.token)
```

Comment:
Is `alias and args.token` even a valid combination?

Reply:
No, this is checked in parser.   User is not supposed to be created manually only via this helper method

Reply:
But if `alias and args.token` is not valid combination, you should either rely on parser to catch that case, or test for it explicitly.  So the check should look like:
```py
    if args.token:
        return User(email=None, host=args.host, password=None,
                    token=args.token)
```
And if you want to guard against `alias and args.token` here as well as in parser, add an explicit assertion before checking for token.

Reply:
Lets trust the parser